amp_conf/htdocs/admin/libraries/Console/Chown.class.php

```diff
@@ -149,6 +149,9 @@ protected function execute(InputInterface $input, OutputInterface $output, $quie
 			$this->modfiles['framework'][] = array('type' => 'execdir',
 			'path' => $AMPBIN,
 			'perms' => 0755);
+			$this->modfiles['framework'][] = array('type' => 'dir',
```

Comment:
type == "dir" (at least according to the comments in the code) does a single directory,
not recursively including the files.  "execdir" does all the files too, and "rdir" does all 
the files, but strips the execute bit.

Isn't 'dir' the right thing to be using here?